src/core/Microsoft.PowerToys.Settings.UI/Views/ImageResizerPage.xaml

```diff
@@ -40,86 +45,136 @@
-                                    <FontIcon Opacity="0.8" Glyph="&#xE74D;"/>
-                                </Button.Content>
-                            </Button>
+                            <AppBarButton x:Name="RemoveButton" 
+                                          Background="Transparent" 
+                                          Command = "{Binding DeleteImageSizeEventHandler, Source={StaticResource ViewModel}}"
+                                          CommandParameter="{Binding Id}"                                          
+                                          Icon="Delete" 
+                                          Width="68" 
+                                          Height="35"/>
                         </StackPanel>
                     </DataTemplate>
                 </ListView.ItemTemplate>
-            </ListView>-->
+            </ListView>
+
+            <StackPanel Orientation="Horizontal">
+                <AppBarButton Icon="Add"
+                              x:Name="AddSizeButton"
+                              IsEnabled="{ Binding Mode=TwoWay, Path=IsEnabled}" 
+                              Label="Add Size"
+                              Margin="{StaticResource SmallTopMargin}"
+                              Command = "{Binding AddImageSizeEventHandler, Source={StaticResource ViewModel}}"
+                          />
 
-            <Button x:Name="AddSizeButton" Content="Add new size" Margin="{StaticResource SmallTopMargin}"/>
+                <AppBarButton Icon="Save"
+                              x:Name="SaveSizeButton"
+                              IsEnabled="{ Binding Mode=TwoWay, Path=IsEnabled}"
+                              Label="Save Sizes"
+                              Margin="{StaticResource SmallTopMargin}"
+                              Command = "{Binding Remove, Source={StaticResource ViewModel}}"
+                          />
+            </StackPanel>
 
             <TextBlock Text="Encoding" 
                        Style="{StaticResource SettingsGroupTitleStyle}"/>
 
             <ComboBox Header="Fallback encoder"
-                          SelectedIndex="1"
+                          SelectedIndex="{ Binding Mode=TwoWay, Path=Encoder}"
                           MinWidth="240"
-                          Margin="{StaticResource SmallTopMargin}">
+                          Margin="{StaticResource SmallTopMargin}"
+                          IsEnabled="{ Binding Mode=TwoWay, Path=IsEnabled}">
+                <ComboBoxItem>PNG Encoder</ComboBoxItem>
                 <ComboBoxItem>BMP Encoder</ComboBoxItem>
-                <ComboBoxItem>GIF Encoder</ComboBoxItem>
                 <ComboBoxItem>JPEG Encoder</ComboBoxItem>
-                <ComboBoxItem>PNG Encoder</ComboBoxItem>
                 <ComboBoxItem>TIFF Encoder</ComboBoxItem>
                 <ComboBoxItem>WMPhoto Encoder</ComboBoxItem>
+                <ComboBoxItem>GIF Encoder</ComboBoxItem>                
```

Comment:
This should also use resource strings.

Reply:
https://github.com/microsoft/PowerToys/pull/2324/commits/6b0ee21b54de0fac62c3d5250ecf143c4dcdfe0f